Dockerfile

```diff
@@ -23,7 +23,9 @@ FROM eclipse-temurin:11-jre-alpine as runner
 ENV FC_LANG en-US LC_CTYPE en_US.UTF-8
 
 # dependencies
-RUN apk upgrade && apk add --update-cache --no-cache bash ttf-dejavu fontconfig curl java-cacerts && \
+RUN apk add -U --update-cache --no-cache bash ttf-dejavu fontconfig curl java-cacerts && \
+    apk -U upgrade && \
+    rm -rf /var/cache/apk/* && \
```

Comment:
No reason to update cache twice - probably better to do something like this: https://github.com/metabase/metabase/pull/15207#pullrequestreview-615421490

Reply:
I just thought about these steps in a better way: I update the cache on the first command without cleaning it, so when I do the upgrade I don't request for the updated package list again. Once all of that is done, I remove the cache :).

Reply:
That's exactly what my command does - just more clearly.